README.md

```diff
@@ -40,14 +40,12 @@ Bash completion requires Git's completion being available.
 
     $ pip install git-imerge
 
-
 or using `setup.py` if you have downloaded the source package locally:
 
     $ python setup.py install
 
-If you prefer, you can use [Homebrew](https://brew.sh) to install `git-imerge`:
-
-    $ brew install git-imerge
+`git-imerge` is also available to install from several distro
+[packages](https://repology.org/project/git-imerge/versions).
```

Comment:
I'm not sure most people would describe a Homebrew formula as a distro package. Perhaps we could reword this slightly to make the breadth of options clearer? For example:

```suggestion
`git-imerge` is also available to install from
[package managers](https://repology.org/project/git-imerge/versions)
on several platforms.
```